docs/how-to/manage-identities.md

```diff
@@ -0,0 +1,145 @@
+
+# How to manage identities
+
+Pebble uses named [identities](../reference/identities.md) to extend access to its [API](../explanation/api-and-clients.md).
+
+For example, let's say you're building a container-based system with one container running Pebble and the workload, and another container in the same user namespace talking to Pebble over the API. You may want to give API access to a non-root user -- that's where identities come in.
+
+By default any UID (user ID) connected to the API socket is a `read` user; UID 0 (root) or the UID of the Pebble server process gives `admin` access.
+
+
+## Add new identities
+
+To extend this to additional users, add named identities using the [`add-identities`](../reference/cli-commands/add-identities.md) command with a YAML file configuring the details. For example, to add a new admin "bob" with UID 42 and a new read user "alice" with UID 2000, prepare this file:
```

Comment:
Suggest a slight change since the flow is broken a little after sectioning out the content:
To extend access to Pebble's API to additional users, add...

Thanks.